tools/depends/target/dav1d/Makefile

```diff
@@ -0,0 +1,39 @@
+include DAV1D-VERSION
```

Comment:
Under what circumstances are `Makefiles` in the `depends/target` folder even used? :thinking: 

I'm not sure if this is needed.

Reply:
the target depends are use for any platform that needs to build all the depends (basically ever platform other than linux).

Reply:
Ok, thanks! Will have to test this in my Windows VM then.